apps/idos-enclave/src/lib/enclave.js

```diff
@@ -309,7 +313,7 @@ export class Enclave {
           keys: () => [],
           reset: () => [],
           configure: () => [mode, theme],
-          storage: () => [humanId, signerAddress, signerPublicKey],
+          storage: () => [humanId, signerAddress, signerPublicKey, expectedUserEncryptionPublicKey],
```

Comment:
Since I have you here, please see if we can get rid of `signerAddress` and `signerPublicKey` which don't seem to be in use anymore — if we can, please do it on a separate PR.